tests/test_perturb.py

```diff
@@ -44,22 +44,29 @@ def test_perturb_insert_space_with_character_size_less_than_two():
 
 
 @pytest.mark.parametrize(
-    "word, expected_result",
-    [("EXTRA", "ETXRA"), ("Hey there", "Hey there"), ("H", "H"),],
+    "word, expected_result", [("THAT", "TAHT"),],
 )
-def test_perturb_swap(word, expected_result):
-    random.seed(42)
-    assert perturb.swap(word) == expected_result
+def test_perturb_shuffle_swap_two(word, expected_result):
+    random.seed(0)
+    assert perturb.shuffle(word, mid=False) == expected_result
+
+
+@pytest.mark.parametrize(
+    "word, expected_result", [("Adversarial", "Aiavrsedarl"),],
```

Comment:
We are missing the test case when ignore=True and we have a " " or length <4
Add this example in parametrize here: ("dog", "dog")

Reply:
@parantak  I think this is the only change we need